### ISSUE TITLE
Change carved pumpkin and jack o lantern to have gravity

### ISSUE THREAD

**Issue by Blacklands** [BTW-Community/BTW-Public#330]
Right now, carved pumpkins are kind of an "uber block" that seems to be too good. The block's only disadvantage is that it cannot be picked up again once placed.

During our search for diamonds we recently realized by accident that carved pumpkins do not burn. We had them with us because we're just drowning in them from farming (one of us once spawned close to the Hardcore Spawn outer radius, walked outside of it by chance while traveling, found wild pumpkins, and brought them home - so we already have them despite not having been to the Nether yet, I don't know if that's intended or just a bunch of luck). So we were just using them as markers now (the bright, orange color is perfect for contrast against the environment).

We randomly tried placing them above the lava lakes that are plentiful deep down there, since they also float (unlike seed-filled pumpkins), and realized that they actually don't burn.
So while just farming for pumpkin seeds you actually also for free get increasing amounts of a block that both floats and does not burn. Isn't that supposed to be something somewhat special? Usually you need clay/slimeballs as mortar for stone to get this. Now we just kind of go caving with a bunch of stacks of these carved pumpkins we have around anyway as leftovers, and build bridges, roofs, close off lava lakes, build staircases down shafts and ravines and the like, and things like that. All without the fear of them burning, unlike with wood.
That seems a bit strong, and also doesn't seem to make much sense.

Is that intended? I know you can't pick them back up, but you just get the for free all the time, I'm sitting on literally like 15 stacks of them right now.
I can also see them being very useful in the Nether (I assume they do not survive Ghast shots, but even so they'd still be quite useful) with these attributes.

I understand that pumpkins are due to some reworking anyway (#49). So maybe in the future this will automatically go away as it won't be this simple anymore to farm massive amounts of pumpkins. Maybe this issue is redundant and should be rolled into #49.
But this feels like a separate issue, since it just feels like pumpkins _should_ burn? I mean, I know IRL they contain a whole bunch of water and are generally fairly tough on the outside, but surely they should still not be able to withstand sustained heat like that.

That they float is also somewhat strange, given that pumpkins with seeds _don't_ float and in fact have a mechanic around them being dropped from some height. Shouldn't carved pumpkins also be subjected to gravity, at the minimum? Maybe also shatter (without dropping anything) if the height is too great? That would also balance out the block somewhat, and maybe would make the burn-immunity fine to keep.

**Comment by Hirachosan** [comment]
I agree that carved pumpkin is a weirdly OP to Post-AAH standards. Over the years it has been overused by many players which considering its an emptied pumkinblovk is kind of weird. I think giving them gravity is a perfectly fine solution to them. And breaking them when dropped similar to regular pumpkins makes sense too.

Jack o lanterns are still a thing though. A gravity block that gives of light might be nice change too, a burn free upgrade to burning netherrack i guess

**Comment by ammoore00** [comment]
I don't think pumpkins should burn, but I'm definitely not opposed to making carved pumpkins and jack o lanterns fall
